ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -823,9 +813,9 @@ fi
 		# Still no MAV_TYPE found
 		if [ $MAV_TYPE == none ]
 		then
-			param set MAV_TYPE 2
+				param set MAV_TYPE 2
```

Comment:
tab issue

Reply:
Fixed and rebased against current master.  I combed through it again and can't spot any other unintended tab insertions.  Let me know if you spot anything else.  Thanks Beat!